app/src/main/java/com/adammcneilly/toa/MainActivity.kt

```diff
@@ -100,6 +113,124 @@ class MainActivity : FragmentActivity() {
 
         val navController = navigationEngine.rememberNavController()
 
+        val windowSizeClass = calculateWindowSizeClass(activity = this)
+
+        val navigationType = getNavigationType(windowSizeClass.widthSizeClass)
+
+        val navigationTabs = listOf(
+            NavigationTab.Home,
+            NavigationTab.Settings,
+        )
+
+        when (navigationType) {
+            NavigationType.BOTTOM_NAVIGATION -> {
+                BottomNavigationContent(
+                    startRoute,
+                    navigationEngine,
+                    navController,
+                    windowSize,
+                    navigationTabs
+                )
```

Comment:
Maybe combine all these properties into a single `MainScreenConfig` data class or similar, so we don't have to tweak every function if this core flow changes.